cypress/integration/main-page.spec.js

```diff
@@ -9,7 +8,5 @@ describe('test main page', () => {
     cy.get('[data-cy=main-page-cars]').should('exist').and('be.visible');
     cy.get('[data-cy=main-page-cars]').children().should('have.length', 12);
   });
-  it('should go to search page', () => {
-    cy.select('brand');
-  });
+ 
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7128486)
```suggestion

```